examples/tableau-infrastructure-modules/environments/staging/backup-bucket/backend.hcl

```diff
@@ -0,0 +1,7 @@
+bucket = "<YOUR BUCKET FOR ENVIRONMENT>"
+dynamodb_table = "<YOUR DYNAMODB TABLE FOR LOCKING>"
+encrypt = true
+key = "tableau-backup/terraform.tfstate"
+region = "<YOUR REGION>"
```

Comment:
We might want to use _ instead of spaces? so they can find and replace?

Reply:
fair point. Will update

Reply:
I've standardised all the values - Should be more consistent and with underscores